.dockerignore

```diff
@@ -1,7 +1,13 @@
 *
-!build/*-runner
-!build/*-runner.jar
-!build/lib/*
-!build/quarkus-app/*
-
```

Comment:
Is it because the build step is done in GitHub actions now?

Reply:
no, b/c of multi stage builds we need to make sure that these are not ignored as they are needed for the secondary stage